server/plugin/utils.go

```diff
@@ -370,6 +372,29 @@ func isValidURL(rawURL string) error {
 	return nil
 }
 
+func buildPluginURL(client *pluginapi.Client, elem ...string) (string, error) {
+	siteURL, err := getSiteURL(client)
+	if err != nil {
+		return "", err
+	}
+
+	redirectURL, err := url.JoinPath(siteURL, append([]string{"foo", Manifest.Id}, elem...)...)
```

Comment:
@hanzei What is "foo" here?

Reply:
Great catch! I highly appreciate the double-checking. Fixed via ad80dd5.